support-frontend/assets/pages/paper-subscription-checkout/components/paperCheckoutForm.test.tsx

```diff
@@ -80,16 +97,12 @@ describe('Newspaper checkout form', () => {
 		});
 
 		renderWithStore(<PaperCheckoutForm />, {
-			// @ts-expect-error -- Type mismatch is unimportant for tests
 			initialState,
 			// @ts-expect-error -- Type mismatch is unimportant for tests
 			store: setUpStore(initialState),
 		});
 	});
 	describe('Payment methods', () => {
-		// it('shows the direct debit option when the currency is GBP and the delivery address is in the UK', () => {
```

Comment:
I think I just forgot to re-enable this test after the recent direct debit incident- could we put it back?